code/game/gamemodes/cult/cult_mode.dm

```diff
@@ -166,6 +166,7 @@ GLOBAL_LIST_EMPTY(all_cults)
 		check_cult_size()
 		cult_objs.study(cult_mind.current)
 		to_chat(cult_mind.current, "<span class='motd'>For more information, check the wiki page: ([GLOB.configuration.url.wiki_url]/index.php/Cultist)</span>")
+		RegisterSignal(cult_mind.current, COMSIG_MOB_STATCHANGE, PROC_REF(check_cult_size))
```

Comment:
Might want to have it check if the cultist died or got revived to avoid doing logic when the cultist only fell asleep or woke up etc.

Reply:
I'm gonna go with the first one, the second one would mean there would be multiple sources of truth, which means the data could fall out of sync unless the code is perfectly maintained.